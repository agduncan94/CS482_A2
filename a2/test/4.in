>seq1
jnknjDEDij=+[];[];[[u8#$%#@$@#DCIRFrf3
>seq2
FNEUFNEIFnefinerofnfifiojf8f9343%$#%{}{][fr[}[}{}RFremfeorpfmefrF{}EF{{E{F}FE{}FP}{}{}[]][
