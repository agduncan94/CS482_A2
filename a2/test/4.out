-25
jnknjDEDi-j=+[];[];[[u8#$%#@$@#DCIRFrf3
-nfif---iojf8f9343-------%--$-#%{}{][f-
